test/box/gh-5928-several-listen-sockets.result

```diff
@@ -0,0 +1,429 @@
+-- test-run result file version 2
+env = require('test_run')
+ | ---
+ | ...
+net_box = require('net.box')
+ | ---
+ | ...
+fio = require('fio')
+ | ---
+ | ...
+fiber = require('fiber')
+ | ---
+ | ...
+test_run = env.new()
+ | ---
+ | ...
+test_run:cmd("create server test with script='box/gh-5928-several-listen-sockets.lua'")
+ | ---
+ | - true
+ | ...
+
+test_run:cmd("setopt delimiter ';'")
+ | ---
+ | - true
+ | ...
+-- borrowed from load_cfg.lua
+function str_split (inputstr, sep)
+    local t = {}
+    for str in string.gmatch(inputstr, "([^"..sep.."]+)") do
+        t[#t + 1] = str:gsub("^%s*(.-)%s*$", "%1")
+    end
+    return t
+end;
+ | ---
+ | ...
+function check_connection(server_addr)
+    local conn = net_box.new(server_addr)
+    if not conn then
+        return false
+    end
+    local rc = conn:ping()
+    conn:close()
+    return rc
+end;
+ | ---
+ | ...
+function prepare_several_listen_uri(default_server_addr, listen_sockets_count)
+    local listen = ""
+    for i = 1, listen_sockets_count do
+        local number = 97 + i - 1
```

Comment:
Why `97 + i -1`? What is that?

Reply:
ascii code, i rename it to make it clear